### ISSUE TITLE
More Default Ox Tethers for AI

### ISSUE THREAD

**Issue by LordDerMoff** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#1013]
Tldr: A default minimum of 3 Ox Tethers per Quarry would benefit the AIs economies

Thanks to the patch, the AI isn't Ox-tethering itself to brain death any more. The only benefit it had, that it used all the stones of their quarries sooner or later. Now, in most cases the quarries are not used at nearly full extent as there are not enough Oxen (or way too far away and inefficient). 

**Therefore, I propose a Compromise:** 
1. The ideal Way: Improving the Placement of Ox-Tethers! Problem: This is probably really complicated and could lead to further bugs
2. The Easy Way: Setting a higher default number of Ox-Tethers per Quarry!
 → BUT is there a default number/minimum per Quarry? (for me, it seems like there are around 2-3 per Quarry, which is kind of weird)

I would argue that with the usual bad locations of the Tethers, the AI would need around 3-4 per Quarry
1. 4 Oxen: could interfere with the rest of the economy (Beer, Population) too much and sometimes an overkill for a quarry
2. 3 Oxen: don't interfere too much and will usually not be enough for a 100% efficient quarry, but still a considerable improvement

**Comment by Russianstory** [comment]
I'd link your offer to my bull tether. https://github.com/UnofficialCrusaderPatch/UnofficialCrusaderPatch/issues/1014


 And I’ll add that it is necessary to make animals, such as cows and bulls, less vulnerable to arrows and bolts for DEFINITELY. And also, to strengthen their lives from blows with melee weapons. Just a dying bull from a stone, or a single arrow is really funny.
